pkg/pub_worker/lib/src/bin/pana_wrapper.dart

```diff
@@ -138,6 +142,30 @@ Future<void> main(List<String> args) async {
   final updatedReport = summary.report?.joinSection(docSection);
   final updatedSummary = summary.change(report: updatedReport);
 
+  // sanity check on pana report size
+  final reportSize =
+      gzip.encode(utf8.encode(json.encode(summary.toJson()))).length;
+  if (reportSize > _reportSizeDropThreshold) {
+    summary = Summary(
+      createdAt: summary.createdAt,
+      runtimeInfo: summary.runtimeInfo,
+      tags: ['has:pana-report-exceeds-size-threshold'],
```

Comment:
Should we do a similar thing here:
https://github.com/dart-lang/pub-dev/blob/a10d45071d0ee399e3b9847fda2dce0dff357683/pkg/pub_worker/lib/src/analyze.dart#L278-L283

Reply:
Not in this PR :D